plugins/telem.go

```diff
@@ -0,0 +1,62 @@
+package plugins
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	otelprom "go.opentelemetry.io/otel/exporters/prometheus"
+	"go.opentelemetry.io/otel/exporters/stdout/stdouttrace"
+	"go.opentelemetry.io/otel/sdk/metric"
```

Comment:
sadly, otel metrics are yet a standard. the implementation is in beta. `Creating Metrics 
The metrics API is currently unstable, documentation TBA.` [https://opentelemetry.io/docs/instrumentation/go/manual/#creating-metrics]

the industry standard is otel for tracing and straight prometheus for metrics. this is a problem that i have faced in the past.

it's unclear if otel metrics will really ever be an industry standard, and so it would be better to grpc middleware prom provider until it has broader adoption.

Reply:
Do you have a link to the alternative?